src/sqlfluff/rules/L014.py

```diff
@@ -11,10 +11,10 @@
 from sqlfluff.rules.L010 import Rule_L010
 
 
-def unquoted_ids_policy_applicable(
+def identifiers_policy_applicable(
     policy: str, parent_stack: Tuple[BaseSegment, ...]
 ) -> bool:
-    """Does `unquoted_identifiers_policy` apply to this segment?"""
+    """Does `(un)quoted_identifiers_policy` apply to this segment?"""
```

Comment:
While not strictly necessary, I wonder if we may want to add:
```
if policy == "none":
    return False
```

That would make the code easier to understand and potentially a bit faster.